mice-and-cats-java/src/main/java/foop/Server.java

```diff
@@ -118,17 +118,18 @@ void runClientReader(Player player) {
                     synchronized (games) {
                         player.setReady(m.ready());
                         if (m.ready() && player.getGame() != null) {
-                            player.getGame().startIfAllReady();
+                            if (player.getGame().startIfAllReady()) {
+                                games.remove(player.getGame().getName());
```

Comment:
What is the purpose of this line?

Reply:
If a game starts we remove it from the list, so that other players don't see it when they click join game

Reply:
ah, I see. thanks